src/main/kotlin/net/atos/client/sd/model/wizard/UnattendedResponseDeclarations.kt

```diff
@@ -2,11 +2,18 @@
  * SPDX-FileCopyrightText: 2024 Lifely
  * SPDX-License-Identifier: EUPL-1.2+
  */
-package net.atos.client.sd.model
+package net.atos.client.sd.model.wizard
```

Comment:
Trivial but I don't like abbreviations in code. I would start refactoring this code and rename the package from `sd` to `smartdocuments`. Just makes the code easier to maintain I feel.